### ISSUE TITLE
phonegap.js is doubled in size when project is created from a cordova github repo

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#71]
```
There is now a constructed cordova.android.js in the framework/assets/js 
directory. In addition to the contents of the other files, it includes some 
common js stuff.

The size increase causes no known functionality problems.

A workaround is to copy framework/assets/js/cordova.android.js to the www 
directory and use that instead of phonegap.js


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 29 Feb 2012 at 4:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 6 Mar 2012 at 4:34
- Added labels: **Milestone-Release1.2.8**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.2.8
```

Original comment by `paul.beu...@gmail.com` on 19 Mar 2012 at 11:35
- Changed state: **Fixed**
